### PR TITLE
Use CockroachDB as app database and dockerize app

### PR DESCRIPTION
**Pull request by atengler**
- add docker-compose for local CockroachDB deployment
- connect to local CockroachDB instance in app config
- tidy compose files
- add bootstrap_env.sh script to simplify starting of support service
- add Dockerfile and entrypoint.sh for app

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+services:
+  roach-ui:
+    image: cockroachdb/cockroach
+    command: start --insecure
+    expose:
+     - "8080"
+     - "26257"
+    ports:
+     - "26257:26257"
+     - "8080:8080"
```
Comment (katyafervent) [inline docker-compose.yaml]: well, we do have keycloak on that port))witch should we change?))

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+version: '3.2'
+
+services:
+  roach-ui:
+    image: cockroachdb/cockroach
+    command: start --insecure
```
Comment (katyafervent) [inline docker-compose.yaml]: this should be changed in production, right? I mean to set up secure connection, right?

**Review by katyafervent — DISMISSED** [review]

File: docker-compose.yaml (hunk 2 above)
Comment (atengler) [reply docker-compose.yaml]: Yes, the production container needs to contain certs which cockroach requires to run in secure mode

File: docker-compose.yaml (hunk 1 above)
Comment (atengler) [reply docker-compose.yaml]: I will change the port here

**Review by katyafervent — APPROVED** [review]

**Pull request merged**